lib/mutant/matcher/config.rb

```diff
@@ -26,7 +26,9 @@ class Config
 
       DEFAULT = new(Hash[anima.attribute_names.map { |name| [name, []] }])
 
-      expression = ->(input) { Mutant::Config::DEFAULT.expression_parser.call(input) }
+      expression = Transform::Block.capture(:expression) do |input|
+        Mutant::Config::DEFAULT.expression_parser.call(input)
```

Comment:
I want function composition as a syntax feature so badly. Miss you `Data.Function.`.``.